tests/CMakeLists.txt

```diff
@@ -19,11 +19,15 @@ endif()
 
 set(TEST_LIBS quda_test)
 
+# custom target for building all tests (useful if QUDA_BUILD_ALL_TESTS is OFF)
+add_custom_target(buildtests)
```

Comment:
I like the funcitonality but wondering if `build tests` is a good name. How about something like `all_tests` or `quda_tests`. 
Buildtests to me sound like this targets builds tests that test whether the build works.